### PR TITLE
Update Roslyn to version 4.8.0-3.23475.6

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Comment by JoeRobich** [comment]
From the CI errors, it seems like source generated documents aren't being returned when requested, completion is using incorrect end of line sequences in empty documents, and completion changed how they return text edits.

File: src/OmniSharp.Roslyn.CSharp/Services/Navigation/SourceGeneratedFileService.cs [hunk 1]
```diff
@@ -20,6 +22,9 @@ public class SourceGeneratedFileService :
         IRequestHandler<UpdateSourceGeneratedFileRequest, UpdateSourceGeneratedFileResponse>,
         IRequestHandler<SourceGeneratedFileClosedRequest, SourceGeneratedFileClosedResponse>
     {
+        private static MethodInfo? CreateDocumentIdFromSerialized => typeof(DocumentId)
```
Comment (333fred) [inline src/OmniSharp.Roslyn.CSharp/Services/Navigation/SourceGeneratedFileService.cs:25]: Should we update the EA for this? Would prefer that over readding reflection.

Comment (JoeRobich) [reply src/OmniSharp.Roslyn.CSharp/Services/Navigation/SourceGeneratedFileService.cs]: @333fred Updated to remove reflection and use EA.

**Review by 333fred — APPROVED** [review]
Make sure to update the title when merging.

**Pull request merged**